### PR TITLE
Create 2017-07-19-NUnit3TestAdapter-3.8.md

### PR DESCRIPTION
**Pull request by OsirisTerje**
News information for release of 3.8 version of the test adapter

### CODE REVIEW

**Comment by ChrisMaddock** [comment]
This looks ok to me. 🙂 

Is it work updating the Downloads page at the same time? 
http://nunit.org/download/

File: _posts/2017-07-19-NUnit3TestAdapter-3.8.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+NUnit3Testadapter, version 3.8 is released.
+
+Major feature is support for .net core /.net standard.
```
Comment (jnm2) [inline _posts/2017-07-19-NUnit3TestAdapter-3.8.md]: Minor nitpick, I think we should follow the official stylization `.NET Core/.NET Standard`.

File: _posts/2017-07-19-NUnit3TestAdapter-3.8.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+NUnit3Testadapter, version 3.8 is released.
```
Comment (jnm2) [inline _posts/2017-07-19-NUnit3TestAdapter-3.8.md]: Nitpick again, sorry; think we should follow the official stylization with a capital A.

File: _posts/2017-07-19-NUnit3TestAdapter-3.8.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Major feature is support for .net core /.net standard. 
+
+See [the release notes](https://github.com/nunit/docs/wiki/Adapter-Release-Notes) for other fixes.
+
+[Download nuget package](https://www.nuget.org/packages/NUnit3TestAdapter/3.8.0)
```
Comment (jnm2) [inline _posts/2017-07-19-NUnit3TestAdapter-3.8.md]: NuGet?

File: _posts/2017-07-19-NUnit3TestAdapter-3.8.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+[Download vsix](https://marketplace.visualstudio.com/items?itemName=NUnitDevelopers.NUnit3TestAdapter)
```
Comment (jnm2) [inline _posts/2017-07-19-NUnit3TestAdapter-3.8.md]: VSIX?

**Pull request merged**